Typescript_TypeORM_Project/src/app.ts

```diff
@@ -17,7 +17,7 @@ const connectDB = new DataSource({
   port: parseInt(process.env.DB_PORT || "5432"),
   entities: ["src/entities/Post.ts", "src/entities/User.ts"],
   logging: false,
-  synchronize: false,
+  synchronize: true,
```

Comment:
This will remove everything from your db and create tables again
With this you might lose your data

Reply:
updated